scripts/install/production/roles/tgt/tasks/main.yml

```diff
@@ -0,0 +1,113 @@
+---
+#Install tgt and its necessary dependencies
+
+- name: Install packages for tgt CentOS installation
+  package: name={{ item }} state=latest
+  with_items:
+   - gcc
+   - cpan
+   - make
+   - firewalld
+  become: true
+  when: ansible_distribution == 'CentOS' or ansible_distribution == 'Red Hat Enterprise Linux'
+
+- name: Change SELinux to permissive for CentOS
+  selinux:
+    policy: targeted
```

Comment:
Permissive is basically disabled with warnings on. No need to disable it.

Reply:
Again, BMI pro not working with permissive. Tested in Kumo.

Reply:
from this page [here](https://wiki.gentoo.org/wiki/SELinux/Tutorials/Permissive_versus_enforcing)
 
> * in permissive mode SELinux does not enforce its policy, but only logs what it would have blocked (or granted)
> * applications that are SELinux-aware might still behave differently with permissive mode than when SELinux is completely disabled

Based on the second point, I'll defer to you on this one. But do you know what selinux aware app we have that needs it disabled? TGT? 

cc: @chemistry-sourabh

Reply:
I don't completely understand the meaning of "selinux aware". I'll have to read on it.